docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -138,9 +178,18 @@ function generateMarkdownFromProperties(
   object.properties.forEach((property) => {
     const defaultValue = property.tags.default?.text?.[0].text;
 
+    let planImg: string;
+    if (property.projects.includes('x-data-grid')) {
+      planImg = '';
+    } else if (property.projects.includes('x-data-grid-pro')) {
+      planImg = ' <span class="plan-pro"></span>';
```

Comment:
I like the idea of having icons here as well!
Can we also link to a pro plan page, as we do for example [here](https://mui.com/components/data-grid/selection/#multiple-row-selection)?
And maybe adding `title="Pro plan"`, so it's clear what that icon means

Reply:
Done

The `title="Pro plan"` could also be interested when we use those icons for doc sections.